main/lsp/wrapper.cc

```diff
@@ -15,7 +16,7 @@ vector<unique_ptr<LSPMessage>> LSPWrapper::getLSPResponsesFor(unique_ptr<LSPMess
     return output->getOutput();
 }
 
-vector<unique_ptr<LSPMessage>> LSPWrapper::getLSPResponsesFor(vector<unique_ptr<LSPMessage>> &messages) {
+vector<unique_ptr<LSPMessage>> LSPWrapper::getLSPResponsesFor(vector<unique_ptr<LSPMessage>> messages) {
```

Comment:
Accepting a vector reference meant that callers didn't have to `move()` the vector, but the vector ended up getting moved later down the line.

I decided to amend the argument to prevent this potential footgun -- especially now that clang in-editor warns about using moved variables!